llvm/lib/Target/SPIRV/SPIRVPrepareFunctions.cpp

```diff
@@ -312,6 +316,54 @@ bool SPIRVPrepareFunctions::substituteIntrinsicCalls(Function *F) {
   return Changed;
 }
 
+bool SPIRVPrepareFunctions::stripConvergenceOperands(Function *F) {
+  DenseSet<Instruction *> to_remove;
```

Comment:
Please correct the variable name according to the llvm coding standard ("The name should be camel case, and start with an upper case letter").

Reply:
Ah yes, still not used to this coding style, thanks!